doc/scenario.md

```diff
@@ -641,6 +653,37 @@ Input facts (set by config):
 - [DEPRECATED] `cartridge_failover` - indicates if eventual failover should be enabled or disabled;
 - `cartridge_failover_params` - failover parameters.
 
+### wait_all_instances_alive
+
+Waits until all cluster instances become alive.
+
+*If `control_instance` is not defined then [set_control_instance](#set_control_instance) will run.*
+
+Input facts (set by role):
+
+- `control_instance` - information about control instance ([more details here](#set_control_instance)).
+
+Input facts (set by config):
+
+- `all_instances_alive_timeout` - time to wait until all instances become alive;
+- `all_instances_alive_delay` - delay to retry instances status check.
+
+### wait_cluster_has_no_issues
+
+Waits until the cluster has no issues.
```

Comment:
won't has?

Reply:
I'm not pretty sure, but my suggestion seems to be OK.